docs/database/database-metrics.md

```diff
@@ -142,7 +142,7 @@ This metric is [recommended][MetricRecommended].
 <!-- semconv metric.db.client.connections.create_time(metric_table) -->
 | Name     | Instrument Type | Unit (UCUM) | Description    |
 | -------- | --------------- | ----------- | -------------- |
-| `db.client.connections.create_time` | Histogram | `ms` | The time it took to create a new connection |
+| `db.client.connections.create_time` | Histogram | `s` | The time it took to create a new connection |
```

Comment:
I am aware that the current recommendation is in seconds, but given the minimal time required to establish a connection—possibly in a tiny fraction of a second—should we change it?

Reply:
I don't think so. Double floating points have enough precision to capture nano-second precision durations until the durations get quite large. I believe you need to be recording something on the order of a few years in nanoseconds before precision is lost.

Backends can easily translate from seconds to ms, us, or ns without loosing precision so I think its best to conform to the recommendation.